src/lando/pushlog/admin.py

```diff
@@ -29,14 +86,48 @@ class PushAdmin(PushLogAdmin):
         "branch",
         "datetime",
         "user",
-        "commits",
-        "tags",
+        # "commits",
+        # "tags",
```

Comment:
Why are these commented out?

Reply:
They are not directly editable anymore, as they have been added to the `exclude` list, and replaced with the Inline. 

So, tl;dr: no longer needed, will remove.